src/main/java/htsjdk/tribble/AbstractFeatureReader.java

```diff
@@ -42,6 +44,8 @@
 
     // the path to underlying data source
     String path;
+    final Function<SeekableByteChannel, SeekableByteChannel> wrapper;
+    final Function<SeekableByteChannel, SeekableByteChannel> indexWrapper;
```

Comment:
I kept them at the same level as path, because any implementations that need to access the wrapper would presumably also need to access the path.

Docs added.